sympy/combinatorics/fp_groups.py

```diff
@@ -514,6 +514,137 @@ def elements(self):
         P, T = self._to_perm_group()
         return T.invert(P._elements)
 
+def subgroup_quotient(G, H, parent_group=None, homomorphism=False):
+    '''
+    Compute the quotient group G/H.
+    The quotient group is computed on a new FreeGroup when
+    `G` is a list of the elements of parent_group.
+
+    Arguments
+    =========
+    G -- A list of `FreeGroupElement`s or an `FpGroup`.
+    H -- A list of `FreeGroupElement`s or an `FpGroup`.
+    parent_group -- A group specified when `G` and `H` are given by a list of generators.
+    homomorphism -- Return a homomorphism whenever `homomorphism` = True
+
+    Returns
+    =======
+    * When `homomorphism = True`, quotient group along with the homomorphism
+    from the quotient to the parent_group whose image is isomorphic to G.
```

Comment:
I've just realised that this doesn't work. In general, there is no well-defined homomorphism from a quotient to the parent group, because each element of the quotient would have to be mapped to multiple elements (everything in the corresponding coset). We can only return a surjective homomorphism from `G` to the quotient. And if we were defining a homomorphism from `parent_group` to the quotient instead, it would have to be a partial one because there would be nowhere to send elements outside of `G`, and that's not possible with `homomorphism`. The best I can think of would be to return two homomorphisms at once when `G` is a list: one an injection from `G` into `parent_group` (what you get from `subgroup()`) and the other the surjection from `G` onto the quotient. What do you think of this?